GestureHandler.js

```diff
@@ -563,11 +563,13 @@ class BaseButton extends React.Component {
   };
 
   render() {
-    const { style, ...rest } = this.props;
+    const { style, rippleColor, ...rest } = this.props;
 
     return (
       <RawButton
         style={[{ overflow: 'hidden' }, style]}
+        rippleColor={processColor(rippleColor)}
+        useRippleColor={!!rippleColor}
```

Comment:
why do we need this new property?

Reply:
why can't we just use `rippleColor` property and if we want the default color then it should be `undefined`?

Reply:
Undefined is casted do zero on Android side. I know that maybe it could be achieved with one property, but I suppose it's clearer and easier solution

Reply:
I think if you change type of rippleColor property in java to `Integer` instead of `int` it should give you `null` in that case:
```
@ReactProp(name = "rippleColor")
public void setRippleColor(ButtonViewGroup view, Integer rippleColor)
```

Reply:
Yes, you're right. I changed it this way.